fe/fe-core/src/main/java/com/starrocks/sql/ast/InsertStmt.java

```diff
@@ -111,6 +111,10 @@ public enum ColumnMatchPolicy {
     // if isFromOverwrite is true, it means that the insert statement is generated by the overwrite statement
     private boolean isFromOverwrite = false;
 
+    private boolean isRewrite = false;
+
+    private boolean rewriteAll = false;
```

Comment:
rewriteall and isRewrite is weird， insert statement do not have rewrite semantic, It's only used for icebergRewriteData. you can new a class like icebergRewriteDataJob and use InsertStmt as a class member.